llarp/link/link_manager.cpp

```diff
@@ -9,27 +10,31 @@
 
 namespace llarp
 {
-  link::Endpoint*
-  LinkManager::GetCompatibleLink(const RouterContact&)
+
+  LinkManager::LinkManager(Router& r)
+      : router{r}
+      , quic{std::make_unique<oxen::quic::Network>()}
+      , tls_creds{oxen::quic::GNUTLSCreds::make_from_ed_keys(
+            {reinterpret_cast<const char*>(router.encryption().data()), router.encryption().size()},
+            {reinterpret_cast<const char*>(router.encryption().toPublic().data()), size_t{32}})}
+      , ep{quic->endpoint(router.local), *this}
+  {}
+
+  std::shared_ptr<link::Connection>
+  LinkManager::get_compatible_link(const RouterContact& rc)
```

Comment:
This function was meant to get an endpoint which can connect to the given RC, but we decided we're just going to have one endpoint and it must be compatible, so this func goes away then.

Reply:
yup its gone